docs/imputation.md

```diff
@@ -23,174 +117,490 @@ This command takes in 4 parameters:
-to the remaining ancestors and the ancestor with the most reads is selected. This is repeated until either the proportion
-of reads mapping to the selected ancestors >= min-coverage or the number ancestors selected equals max-ancestors. If a
-name is provided for the likely-ancestor-file, the chosen ancestors are written to that file in the order they were chosen
-along with the cumulative coverage, which is the proportion of reads mapping to each ancestor and the previously chosen ones.
+| Parameter name         | Description                                                                                                        | Default value |
+|------------------------|--------------------------------------------------------------------------------------------------------------------|---------------|
+| `--max-hap-proportion` | Only k-mers mapping to less than or equal to maxHapProportion of haplotypes in a reference range will be retained. | `0.75`        |
+| `--max-arg-length`     | The maximum argument length for a call to the [AGC program](https://github.com/acoleman2000/agc).                  | `200000`      |
 
-### Required Parameters
+> [!TIP]
+> If you get an error caused by a call to AGC being too long,
+> try reducing the `--max-arg-length` value.
 
-**--path-keyfile** or **--read-file** but not both. **--path-keyfile** is the name and path of a keyfile that contains a list of the 
-read mapping files or read files (fastq). The keyfile must have two columns labeled sampleName and filename. 
-Any additional columns will be ignored. SampleName must be unique.
-Additionally, if an output hvcf for a sample name already exists in the output directory, the sample will be skipped.
-See the **Input Files** section above for more detail.
+The following optional parameters affect how k-mers are pre-filtered 
+to determine which are used for indexing. They would only need to be 
+adjusted if the number of k-mers in the index is too low or too high:
 
-**--hvcf-dir**: The directory containing the hvcf used to build the haplotype graph used for imputation.
+| Parameter name  | Description                                                                                                                                                               | Default value |
+|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|
+| `--hash-mask`   | In conjunction with `--hash-filter`, used to mask k-mers for filtering. Default uses only the last k-mer nucleotide. **Only change this if you know what you are doing.** | `3`           |
+| `--hash-filter` | Only hashes that pass the filter ((hashValue and hashMask) == hashFilter) will be considered. **Only change this value unless you know what you are doing.**              | `1`           |
```

Comment:
Change to either "Don't change this value unless you know what you are doing" or "only change this value if you know what you are doing"

Reply:
This has been updated